apps/sms/test/marionette/lib/views/new-message/accessors.js

```diff
@@ -14,6 +14,9 @@ var SELECTORS = Object.freeze({
   subjectInput: '.subject-composer-input',
   sendButton: '#messages-send-button',
   attachButton: '#messages-attach-button',
+  headerActionButton: '#messages-options-button',
```

Comment:
I tried looking at the index.html file for the sms app. Couldn't find any action-button class in it :disappointed: